ide/coqide/session.ml

```diff
@@ -506,27 +561,22 @@ let build_layout (sn:session) =
       ~callback:(fun () -> if sn.buffer#modified
         then img#set_stock `SAVE
         else img#set_stock `YES) in
-  (* There was an issue in the previous implementation for setting the
-     position of the handle. It was using the size_allocate event but
-     there is an issue with size_allocate. G. Melquiond analyzed that
-     at starting time, the size_allocate event is only issued in
-     Layout phase of the gtk loop so that it is actually processed
-     only in the next iteration of the event-update-layout-paint loop,
-     after the user does something and trigger an effective new event
-     (see #10578). So we preventively enforce an estimated position
-     for the handles to be used in the very first initializing
-     iteration of the loop *)
-  let () =
-    (* 14 is the estimated size for vertical borders *)
-    let estimated_vertical_handle_position = (fst !window_size - 14) / 2 in
-    (* 169 is the estimated size for menus, command line, horizontal border *)
-    let estimated_horizontal_handle_position = (snd !window_size - 169) / 2 in
-    if estimated_vertical_handle_position > 0 then
-      eval_paned#set_position estimated_vertical_handle_position;
-    if estimated_horizontal_handle_position > 0 then
-      state_paned#set_position estimated_horizontal_handle_position in
+
```

Comment:
This is the main change in the coqide codebase that worries me, wasn't this submitted as a different PR?

I don't recall the outcome of the discussion, but I'm wary folding this kind of unrelated modifications in this PR.

cc: @coq/coqide-maintainers

Reply:
I reversed this.  We earlier decided not to make the change in the PR, #14536.